lmod_proxy/static/site/js/site.js

```diff
@@ -0,0 +1,129 @@
+$(document).ready(function(){
+
+  // Setup nice click handling for gradebook
+  $("#gradebook").click(function() {
+    if(!$(this).hasClass("selected")) {
+      $(this).select();
+      $(this).addClass("selected");
+    }
+  });
+  $("#gradebook").blur(function() {
+    if($(this).hasClass("selected")) {
+      $(this).removeClass("selected");
+    }
+  });
+
+
+  function post_call(data) {
+    $('#error').hide(100);
+    console.log(data)
+    $.post(location.pathname, data)
+      .done(function(data, status){
+        console.log(data);
+        console.log(status);
```

Comment:
Any need for these console.logs? It seems that the convenience of network tools in modern browsers should eliminate the need.